intro.md

```diff
@@ -1,5 +1,5 @@
 # Home 
-*Author: Ahmed Ratnani*
```

Comment:
We should probably keep the authors' names in this first page of the "book". To this end we can list all contributors in an AUTHORS.md file, the contents of which could be displayed here. Ahmed should be the first author as he is the one who started the project.

Reply:
I've just added an `AUTHORS` file and now it looks like this: ![intro](https://github.com/user-attachments/assets/d27a7488-6c76-4df3-b0e2-18a1924fabd5)

Unfortunately the `AUTHORS` file can't be written as one-line per author. Doing so renders the list of authors in a single line (without delimiters). Maybe there's a workaround, but so far I haven't found any solution from the [MyST docs](https://myst-parser.readthedocs.io/en/latest/faq/index.html#include-rst-files-into-a-markdown-file).

Reply:
I see. Then I suggest that we leave only Ahmed as author here, and add a link to the complete AUTHORS file, maybe in the footnote. How does that sound to you?

Reply:
Sounds good! I've already made the changes:

![image](https://github.com/user-attachments/assets/88a561cc-a7c5-4def-b488-9dcb698ce38e)
![image](https://github.com/user-attachments/assets/8b9a5c78-00ae-4649-a73c-3d2dba64d860)

Reply:
That looks great!